timesketch/api/v1/resources/scenarios.py

```diff
@@ -92,84 +110,116 @@ def get(self, sketch_id):
     def post(self, sketch_id):
         """Handles POST request to the resource.
 
-        This resource creates a new scenario for a sketch based on a template.
-        Templates are defined in SCENARIOS_PATH.
+        This resource creates a new scenario for a sketch based on a DFIQ template.
 
         Returns:
             A JSON representation of the scenario.
         """
         sketch = Sketch.query.get_with_acl(sketch_id)
+        dfiq = load_dfiq_from_config()
```

Comment:
Small (optional) nit: You could save on this operation in cases where the sketch does not exist or there is no access when you load the dfiq after the sketch error messages.